src/version.h

```diff
@@ -6,4 +6,9 @@
 /* API versions. */
 #define REDISAI_LLAPI_VERSION 1
 
+/* This is the major / minor encver, to be used as
+   argument to RM_CreateDataType, which expects
+   encver to be < 1024 */
+static const long long RAI_ENC_VER_MM = RAI_ENC_VER == 999999 ? 1023 : RAI_ENC_VER / 100;
```

Comment:
As far as I understand, current (master) RAI_ENC_VER is 0? 
How do you plan to maintain updates to RAI_ENC_VER in the objects RDB decoders?

Reply:
Current master is `RAI_ENC_VER` `999999`, while for 1.0 `RAI_ENC_VER` is `10000` (major `1`, minor `00`, patch `00`).

Reply:
ok
So there is a need to update a minor version once this update is merged, right?
For now, the decoding of objects from RDB is checking if RAI_ENC_VER > 100 for the new partitioned decode, otherwise it will decode as a single buffer. This is ok for now since there are only two "official" versions. Once there will be additional modifications in encoding/decoding objects to RDB I suggest moving to a dedicated decoding function per encoding version.

Reply:
Yes, if we release this will need to be a minor (1.1.0)